docs/director/default-eventing-for-application.md

```diff
@@ -26,3 +25,66 @@ The Director API offers a mutation that allows for assigning the default Runtime
 The Directors API offers a mutation that allows for deleting the assignment of the default Runtime for the Application eventing. The `deleteDefaultEventingForApplication(appID: String!): ApplicationEventingConfiguration!` mutation deletes the `{APPLICATION_ID}/defaultEventing = true` label from the labeled Runtime.
 
 >**NOTE:** After deleting the `{APPLICATION_ID}_defaultEventing = true` label and querying for the **eventingConfiguration** for the Application, the Director API determines the default Runtime for the Application once again.
+
+## Troubleshooting
+
+### Application eventingConfiguration returns empty defaultURL
+
+It is possible for the Director to return empty `defaultURL` property for the Application eventing configuration in certain circumstances:
+- the Application does not have a scenario assigned
+- the Application have scenario assigned, but the scenario does not have any Runtimes assigned
+- the default Runtime determined by the assignment is not fully provisioned or connected to the Compass
+- the Runtime does not have the label for event service URL assigned or the label value is empty
+
+1. Verify the Application.
+
+The following GraphQL snippet queries the application by its ID. The response contains application `id`, `name`, `labels` collection, and `defaultURL` for eventing. Please ensure that the Application belongs to at least one scenario which also has runtime assigned.
+
+```graphql
+query {
+  application(id: "{APPLICATION_ID}") {
+    id
+    name
+    labels
+    eventingConfiguration {
+      defaultURL
+    }
+  }
+}
+```
+
+2. Verify the default Runtime assigned for Application eventing configuration
```

Comment:
`eventing configuration` or `eventingConfiguration`? Please be consistent with your approach and apply it in the whole document.